### ISSUE TITLE
TrainLoss decrease but ValLoss doesn't

### ISSUE THREAD

**Issue by SiruLiu645** [lee-zq/3DUNet-Pytorch#40]

**Comment by SiruLiu645** [comment]
I use this code in my own wholeheart substructure segmentation，but the train loss decrease faster while valloss doesn't，how to solve this problem？

**Comment by AlloyMei** [comment]
> I use this code in my own wholeheart substructure segmentation，but the train loss decrease faster while valloss doesn't，how to solve this problem？

hello, I encountered the same problem, have you solved it?
